02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -6,3 +6,67 @@ output:
     toc: true
     toc_float: true
 ---
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to microarray technology](#introduction-to-microarray-technology)
+  - [Microarray data **strengths**:](#microarray-data-strengths)
+  - [Microarray data **limitations**:](#microarray-data-limitations)
+- [About quantile normalization](#about-quantile-normalization)
+- [More resources on microarray technology:](#more-resources-on-microarray-technology)
+  - [Why doesn't the gene I care about show up in this microarray dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-microarray-dataset)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to microarray technology
+
+Microarrays measure gene expression using chips filled with probes designed to hybridize to RNA samples that have been reverse transcribed to cDNA. 
+Bound probes give off fluorescence the intensity of which indicates the number of probes bound and therefore the relative quantity of the transcript the probe is designed for. 
+
+<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/13fd8c2d8469f2ac35e1608b4a952a5f5fe9698c/components/figures/microarray-overview.jpg" width=400>  
+
+[@microarray-video]
```

Comment:
<img width="325" alt="Screen Shot 2020-10-05 at 9 07 53 AM" src="https://user-images.githubusercontent.com/23458084/95083472-906b5800-06ea-11eb-872f-2d345db7fe56.png">

This is what it looks like on my end. Do we not like this? What if I put a `Image from: @microarray-video`. 

I don't want to link the resource itself right here because I don't consider it the most helpful video (it's probably too basic for this audience and also half talks about DNA microarrays).

Reply:
Oh, yeah, that is what it looks like. I thought you were trying to link to the video. Fine as is.